client/app/templates/application.hbs

```diff
@@ -1,5 +1,19 @@
-{{page-title "Client"}}
+{{page-title (t "page_titles.candidates")}}
```

Comment:
Leverage `ember-intl` to deliver all UI text as translation values (from `en-us.json`).